recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -1,19 +1,20 @@
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -1,14 +1,14 @@
+@@ -1,14 +1,15 @@
 -project(basisu)
 -
  cmake_minimum_required(VERSION 3.0)
 -option(BUILD_X64 "build 64-bit" TRUE)
 -option(STATIC "static linking" FALSE)
 -option(SSE "SSE 4.1 support" FALSE)
 +
-+project(libbasisu VERSION 1.15.0 LANGUAGES CXX)
++project(basisu VERSION 1.15.0 LANGUAGES CXX)
 +
 +option(SSE4 "SSE4 4.1 support" FALSE)
  option(ZSTD "ZSTD support for KTX2 transcoding/encoding" TRUE)
 +option(BASISU_NO_ITERATOR_DEBUG_LEVEL "Change Iterator debug level" FALSE)
 +option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" OFF)
++option(TRANSCODER_ONLY "Build only transcoder part of basisu library")
```

Comment:
```suggestion
+option(TRANSCODER_ONLY "Build only transcoder part of basisu library" FALSE)
```

Reply:
No, no, it should be quite opposite. When the `TRANSCODER_ONLY` option is TRUE, only the transcoder part of library is built (this is custom option), but not the whole library (with the tool) as the upstream does. However I agree there's missing default option FALSE, so the whole library will be built as before.